polysync/encoder.hpp

```diff
@@ -5,27 +5,32 @@
 #include <boost/hana.hpp>
 #include <boost/endian/arithmetic.hpp>
 
-#include <polysync/plog/core.hpp>
 #include <polysync/descriptor.hpp>
 #include <polysync/logging.hpp>
 #include <polysync/exception.hpp>
+#include <polysync/compound_types.hpp>
 
-namespace polysync { namespace plog {
+namespace polysync {
 
 namespace hana = boost::hana;
 namespace endian = boost::endian;
 using logging::severity;
 
-class encoder {
+class Encoder
+{
 public:
 
-    encoder(std::ostream& st) : stream(st) {}
+    Encoder( std::ostream& st ) : stream(st)
+    {
```

Comment:
Can't remember if this made it into the style guide, but for empty constructors I think we've been using `{ /* Empty */ }`